packages/foam-core/types/utils.d.ts

```diff
@@ -1 +1,3 @@
 declare module 'remark-wiki-link';
+
+declare module 'dashify';
```

Comment:
Does this mean there's no typescript definitions for this module?

Reply:
There is! I have added the `@types/dashify` as dep.